butterknife-annotations/src/main/java/butterknife/BindView.java

```diff
@@ -13,9 +13,28 @@
  * <pre><code>
  * {@literal @}BindView(R.id.title) TextView title;
  * </code></pre>
+ *
+ * If used in a library project where the fields are not final, use the other attributes:
+ * <pre><code>
+ * {@literal @}BindView(idName = "title") TextView title;
```

Comment:
Android Studio sometimes refactors names if it finds them in strings, I haven't tried with this method but that might apply. However, if this string isn't changed then assuming you don't have anything else with the same ID (which I think would be changed with an IDE refactor anyway?) then this will cause a compile-time error as the ID will no longer exist.